src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -107,6 +109,12 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .SetDefault(false)
                 .Callback(s => UseHttps = s);
 
+            Parser
+                .Setup<bool>("useDefaultCert")
```

Comment:
actually would you mind updating this to `use-default-cert` instead? I know this command isn't very consistent since it has `useHttps`, but that was one of the first flags added here and I think everything else started using `kebab-case`